README.md

```diff
@@ -57,11 +57,16 @@ and start the latest build, locally, like `docker run -p 18081:8080 -it aerogear
 
 ## Releasing the UnifiedPush Server
 
-The content of the [Release Process](https://github.com/aerogear/collateral/wiki/Release-Process-(Java)) is valid for this project as well. However, to build the `distribution` bundle, you need to include these profiles:
+The content of the [Release Process](https://github.com/aerogear/collateral/wiki/Release-Process-(Java)) is valid for this project as well. However, to build the full `distribution` bundle, you need to fire off the release like:
 
 ```
-mvn release:GOAL -Pdist,test
+## prepare the release and define the TAG and adjust the versions:
+mvn release:prepare -Dtag=x.y.z.Final -Darguments=-Dgpg.passphrase=$MY_SECRET_PASS_PHRASE -Pdist,test
+
```

Comment:
The tag needs to be deleted and recreated from the previous commit after that step right?

Reply:
yes, that's mentioned in the real release process document, the sentence before this snippet does actually link to it